kvmagent/kvmagent/plugins/vmware_v2v_plugin.py

```
@@ -607,8 +608,8 @@ def _check_str_in_file(self, fname, txt):
     @in_bash
     def _get_qcow2_sizes(self, path):
         cmd = "%s --output=json '%s'" % (qemu_img.subcmd('info'), path)
-        _, output = commands.getstatusoutput(cmd)
-        return long(json.loads(output)['actual-size']), long(json.loads(output)['virtual-size'])
+        _, output = subprocess.getstatusoutput(cmd)
```

Ruff (0.15.2)

[error] Starting a process with a shell, possible injection detected

(S605)